projects/projects.md

```diff
@@ -1,2 +1,2 @@
-1.  [Bitcoin](/projects/bitcoin.md)
-2.  [Ethereum](/projects/ethereum.md)
+1.  [Bitcoin](http://base.keycrypto.vc/projects/bitcoin.html)

```

Comment:
It doesn't work with relative url. Point is that somehow /projects.md turns to html but projects/bitcoin.md doesn't